### ISSUE TITLE
Multifile export -import

### ISSUE THREAD

**Issue by c0ntradicti0n** [keycloak/keycloak#17436]
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

import-export

### Describe the bug

When exporting to multiple files with:

`/opt/keycloak/bin/kc.sh export --dir /opt/keycloak/data/import/ --realm project --users different_files`

and importing again, the users file is tried to imported BEFORE the rest of the realm. And it fails with error not importing the users, because service accounts need services registered before.

This could easily fixed by setting the order of the files. It's not clear, how to manually define that order, prefixing with numbers does also not work.

### Version

20.0.3

### Expected behavior

Importing users

### Actual behavior

```
2023-03-03 11:40:43,187 WARN  [org.keycloak.services] (main) KC-SERVICES0005: Unable to import realm project from file /opt/keycloak/bin/../data/import/project-users-0.json.: java.lang.RuntimeException: Unable to find client specified for service account link. Client: backend
2023-03-03T11:40:43.195693892Z 	at org.keycloak.storage.datastore.LegacyExportImportManager.createUser(LegacyExportImportManager.java:883)
2023-03-03T11:40:43.195702123Z 	at org.keycloak.storage.datastore.LegacyExportImportManager.importRealm(LegacyExportImportManager.java:414)
2023-03-03T11:40:43.195708632Z 	at org.keycloak.models.utils.RepresentationToModel.importRealm(RepresentationToModel.java:126)
2023-03-03T11:40:43.195714803Z 	at org.keycloak.services.managers.RealmManager.importRealm(RealmManager.java:570)
2023-03-03T11:40:43.195720766Z 	at org.keycloak.services.managers.RealmManager.importRealm(RealmManager.java:503)
2023-03-03T11:40:43.195745843Z 	at org.keycloak.services.resources.KeycloakApplication.importRealm(KeycloakApplication.java:304)
2023-03-03T11:40:43.195753080Z 	at org.keycloak.services.resources.KeycloakApplication.importRealms(KeycloakApplication.java:280)
2023-03-03T11:40:43.195759036Z 	at org.keycloak.services.resources.KeycloakApplication.bootstrap(KeycloakApplication.java:213)
2023-03-03T11:40:43.195764837Z 	at org.keycloak.quarkus.runtime.integration.jaxrs.QuarkusKeycloakApplication.startup(QuarkusKeycloakApplication.java:43)
2023-03-03T11:40:43.195770627Z 	at org.keycloak.quarkus.runtime.integration.QuarkusLifecycleObserver.onStartupEvent(QuarkusLifecycleObserver.java:37)
2023-03-03T11:40:43.195776682Z 	at org.keycloak.quarkus.runtime.integration.QuarkusLifecycleObserver_Observer_onStartupEvent_b0e82415b143738dc1f986a5fa4668e83d0a5dea.notify(Unknown Source)
2023-03-03T11:40:43.195783358Z 	at io.quarkus.arc.impl.EventImpl$Notifier.notifyObservers(EventImpl.java:326)
2023-03-03T11:40:43.195789205Z 	at io.quarkus.arc.impl.EventImpl$Notifier.notify(EventImpl.java:308)
2023-03-03T11:40:43.195794442Z 	at io.quarkus.arc.impl.EventImpl.fire(EventImpl.java:76)
2023-03-03T11:40:43.195799545Z 	at io.quarkus.arc.runtime.ArcRecorder.fireLifecycleEvent(ArcRecorder.java:131)
2023-03-03T11:40:43.195804672Z 	at io.quarkus.arc.runtime.ArcRecorder.handleLifecycleEvents(ArcRecorder.java:100)
2023-03-03T11:40:43.195810255Z 	at io.quarkus.deployment.steps.LifecycleEventsBuildStep$startupEvent1144526294.deploy_0(Unknown Source)
2023-03-03T11:40:43.195815867Z 	at io.quarkus.deployment.steps.LifecycleEventsBuildStep$startupEvent1144526294.deploy(Unknown Source)
2023-03-03T11:40:43.195822872Z 	at io.quarkus.runner.ApplicationImpl.doStart(Unknown Source)
2023-03-03T11:40:43.195827577Z 	at io.quarkus.runtime.Application.start(Application.java:101)
2023-03-03T11:40:43.195832155Z 	at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:110)
2023-03-03T11:40:43.195836723Z 	at io.quarkus.runtime.Quarkus.run(Quarkus.java:69)
2023-03-03T11:40:43.195841242Z 	at org.keycloak.quarkus.runtime.KeycloakMain.start(KeycloakMain.java:103)
2023-03-03T11:40:43.195846129Z 	at org.keycloak.quarkus.runtime.cli.command.AbstractStartCommand.run(AbstractStartCommand.java:37)
2023-03-03T11:40:43.195850957Z 	at picocli.CommandLine.executeUserObject(CommandLine.java:1939)
2023-03-03T11:40:43.195855434Z 	at picocli.CommandLine.access$1300(CommandLine.java:145)
2023-03-03T11:40:43.195859945Z 	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
2023-03-03T11:40:43.195865435Z 	at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
2023-03-03T11:40:43.195871164Z 	at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)
2023-03-03T11:40:43.195876558Z 	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
2023-03-03T11:40:43.195892219Z 	at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
2023-03-03T11:40:43.195897988Z 	at picocli.CommandLine.execute(CommandLine.java:2078)
2023-03-03T11:40:43.195903531Z 	at org.keycloak.quarkus.runtime.cli.Picocli.parseAndRun(Picocli.java:93)
2023-03-03T11:40:43.195909497Z 	at org.keycloak.quarkus.runtime.KeycloakMain.main(KeycloakMain.java:89)
2023-03-03T11:40:43.195914655Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2023-03-03T11:40:43.195920726Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2023-03-03T11:40:43.195925981Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2023-03-03T11:40:43.195931999Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2023-03-03T11:40:43.195937289Z 	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.doRun(QuarkusEntryPoint.java:61)
2023-03-03T11:40:43.195943350Z 	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.main(QuarkusEntryPoint.java:32)
2023-03-03T11:40:43.195948638Z 
2023-03-03T11:40:45.359249082Z 2023-03-03 11:40:45,358 INFO  [org.keycloak.services] (main) KC-SERVICES0004: Imported realm tbdk from file /opt/keycloak/bin/../data/import/project-realm.json.
2023-03-03T11:40:45.593851438Z 
```

### How to Reproduce?

Export and import with multifile

### Anything else?

Thanks

**Comment by ahus1** [comment]
Thanks for reporting this. I assume you're running

```
bin/kc.[sh|bat] start --import-realm ...
```

to import the data. 

I had a look, and it seems to me that this has been addressed as part of issue #16281 which has been released as part of Keycloak 21.

Could you please give it a try with that release and let me know if it works as expected for you?

**Comment by c0ntradicti0n** [comment]
Ah, I checked it and yeah, it works now! I was not up to date with the newest release those days before.

Thank you very much!
